libs/neo4j/langchain_neo4j/chains/graph_qa/cypher.py

```diff
@@ -120,6 +146,8 @@ def filter_func(x: str) -> bool:
             ",".join(formatted_rel_props),
             "The relationships are the following:",
             ",".join(formatted_rels),
+            "The vector indexes are the following:",
+            ",".join(formatted_vector_indexes),
```

Comment:
Similarly, please write some tests to test this under libs/neo4j/tests/unit_tests/chains/test_graph_qa.py